src/go/rdctl/pkg/factoryreset/factory_reset_windows.go

```
@@ -14,58 +14,57 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package reset implements factory reset for Windows.
-package reset
+package factoryreset
 
 import (
-	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path"
 	"strings"
-	"unsafe"
+	"syscall"
 
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/directories"
 	"github.com/sirupsen/logrus"
-	"golang.org/x/sys/windows"
+	"golang.org/x/text/encoding/unicode"
 )
 
-const (
-	appName = "rancher-desktop"
-)
+func PowershellKillWindows() {
+	cmd := exec.Command("taskkill", "/IM", "Rancher Desktop", "/T", "/F")
+	cmd.SysProcAttr = &syscall.SysProcAttr{CreationFlags: CREATE_NO_WINDOW}
+	cmd.Run()
+}
 
-// Factory reset deletes any Rancher Desktop user data.
-func FactoryReset(keepSystemImages bool) error {
-	dirs, err := getDirectoriesToDelete(keepSystemImages)
+func deleteWindowsData(keepSystemImages bool, appName string) error {
+	dirs, err := getDirectoriesToDelete(keepSystemImages, appName)
 	if err != nil {
 		return err
 	}
 	for _, dir := range dirs {
 		logrus.WithField("path", dir).Trace("Removing directory")
 		if err := os.RemoveAll(dir); err != nil {
-			if !errors.Is(err, os.ErrNotExist) {
-				return fmt.Errorf("could not remove %s: %w", dir, err)
-			}
+			logrus.Errorf("Problem trying to delete %s: %s\n", dir, err)
```

We now continue (swallowing the error). That's probably fine.